integration-tests/src/test/java/tachyon/master/lineage/LineageMastertIntegrationsTest.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.lineage;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
+
+import tachyon.Constants;
+import tachyon.IntegrationTestConstants;
+import tachyon.LocalTachyonClusterResource;
+import tachyon.TachyonURI;
+import tachyon.client.TachyonStorageType;
+import tachyon.client.UnderStorageType;
+import tachyon.client.file.FileOutStream;
+import tachyon.client.file.options.OutStreamOptions;
+import tachyon.client.lineage.LineageMasterClient;
+import tachyon.client.lineage.TachyonLineageFileSystem;
+import tachyon.conf.TachyonConf;
+import tachyon.heartbeat.HeartbeatContext;
+import tachyon.heartbeat.HeartbeatScheduler;
+import tachyon.job.CommandLineJob;
+import tachyon.job.JobConf;
+import tachyon.master.lineage.meta.LineageFileState;
+import tachyon.thrift.LineageInfo;
+
+/**
+ * Integration tests for the lineage module.
+ */
+public final class LineageMastertIntegrationsTest {
+  private static final int BLOCK_SIZE_BYTES = 128;
+  private static final long WORKER_CAPACITY_BYTES = Constants.GB;
+  private static final int QUOTA_UNIT_BYTES = 128;
+  private static final int BUFFER_BYTES = 100;
+
+  @Rule
+  public LocalTachyonClusterResource mLocalTachyonClusterResource =
+      new LocalTachyonClusterResource(WORKER_CAPACITY_BYTES, QUOTA_UNIT_BYTES, BLOCK_SIZE_BYTES,
+          Constants.USER_FILE_BUFFER_BYTES, String.valueOf(BUFFER_BYTES),
+          Constants.WORKER_DATA_SERVER, IntegrationTestConstants.NETTY_DATA_SERVER);
```

Comment:
Can you add the `Constants.USER_LINEAGE_ENABLED, "true"` setting here instead of before each test?